torch/nn/functional.py

```diff
@@ -1160,6 +1160,10 @@ def softsign(input):
 
 softplus = _add_docstr(torch._C._nn.softplus, r"""
 softplus(input, beta=1, threshold=20) -> Tensor
+
+Applies element-wise, the function :math:`\text{Softplus}(x) = \frac{1}{\beta} * \log(1 + \exp(\beta * x))`
```

Comment:
Should we mention `threshold` at all? It's not 100% true that we are applying this function; Softplus applies this function  until the values exceed the threshold

Reply:
Good point! removed!

Reply:
this argument is not supported in F.softplus? or is it just not intended for public use?

Reply:
Sorry, I was suggesting that we add `threshold` to the formula or write a line about it explaining how `\text{Softplus}(x) = \frac{1}{\beta} * \log(1 + \exp(\beta * x))` is only true until `threshold`

Reply:
oh i see. sorry. i'll update.

Reply:
Updated!